packages/stencil-component/src/components/read-along-component/read-along.tsx

```diff
@@ -727,73 +899,125 @@ export class ReadAlongComponent {
    */
   componentDidLoad() {
 
-    this.processed_alignment = parseSMIL(this.alignment)
-    this.assetsStatus.SMIL = Object.keys(this.processed_alignment).length ? LOADED : ERROR_LOADING
 
     // load basic Howl
     this.audio_howl_sprites = new Howl({
       src: [this.audio],
       preload: true,
       onloaderror: this.audioFailedToLoad.bind(this),
-      onload: this.audioLoaded.bind(this)
+      onload: this.audioLoaded.bind(this),
+      onend: this.audioEnded.bind(this)
+
 
     })
     // Once loaded, get duration and build Sprite
     this.audio_howl_sprites.once('load', () => {
-
```

Comment:
This seems like only pr mostly a whitespace change, it may create a nasty conflict when merged with dev.readalong-format